ams/optimizer/common/src/main/java/com/netease/arctic/optimizer/common/OptimizerExecutor.java

```diff
@@ -100,7 +118,16 @@ private boolean ackTask(OptimizingTask task) {
   }
 
   protected OptimizingTaskResult executeTask(OptimizingTask task) {
-    return executeTask(getConfig(), getThreadId(), task, LOG);
+    OptimizingTaskResult result = executeTask(getConfig(), getThreadId(), task, LOG);
+    // add optimizer flink runtime info, including application_id, tm_id, host
+    StringBuilder sb = new StringBuilder();
+    if (StringUtils.isNotEmpty(result.getErrorMessage())) {
```

Comment:
I recommend replacing it with `com.google.common.base.Strings#isNullOrEmpty`.

Reply:
done